pkg/restclient/volume.go

```diff
@@ -69,6 +69,12 @@ func (c *Client) VolumeSet(volname string, req api.VolOptionReq) error {
 	return err
 }
 
+// GlobalOptionSet sets cluster level options
+func (c *Client) GlobalOptionSet(req api.GlobalOptionReq) error {
+	url := fmt.Sprintf("/v1/cluster/options")
+	return c.post(url, req, http.StatusOK, nil)
+}
+
```

Comment:
Also need `GlobalOptions()`

Reply:
If you mean GlobalOptionsGet() do you think it makes sense to add this part after the patch on volume get support is merged?

Reply:
Sure, makes sense.